tests/tests_integration/test_api/test_simulators/seed/data.py

```diff
@@ -0,0 +1,216 @@
+# This file contains the data used to seed the test environment for the simulator tests
+
+data_set_id = 1521375514069
```

Comment:
I agree this should no be hardcoded as an internal id. it should be an external id + lookup